configuration/rules-dsl.md

```diff
@@ -301,7 +301,7 @@ The following table summarizes the impact of the two manipulator commands on the
 **Beware:**
 In most cases, a rule with a trigger of `received update` will fire following the command `sendCommand` as:
 - openHAB auto-updates the status of Items for which the item definition does not contain `autoupdate="false"`
-- the Thing sends an status update to the Item.
+- the Thing sends a status update to the Item.
```

Comment:
Doesn't have to happen in this PR but if you'd be up for it here or in an extra PR: please help improve these "Beware" lines of the document. I read it multiple times and it's just impossible to grasp, even for me who knows what it is trying to say ^^